src/services/completions.ts

```diff
@@ -3051,4 +3054,9 @@ namespace ts.Completions {
             addToSeen(seenModules, getSymbolId(sym)) &&
             checker.getExportsOfModule(sym).some(e => symbolCanBeReferencedAtTypeLocation(e, checker, seenModules));
     }
+
+    function isDeprecated(symbol: Symbol, checker: TypeChecker) {
+        const declarations  = skipAlias(symbol, checker).declarations;

```

Comment:
I think we should move `isDeprecatedDeclaration` thats already in symbolDisplay.ts to utilities and use that...